### PR TITLE
create visualization graph

### PR DESCRIPTION
**Pull request by YukaUU**
Add visualization panel to viewer page.
## Summary

Click the icon to the left of the toolbar's tutorial to open the side menu on the right side.
There you can see a graph visualizing the annotation information
It shows the relationship between human:annotation and zooming.
all
## Motivation

To visualize pathologist annotation information
## Testing

The graph and the results with collection:Mark.
Annotation:0
d0
d0_log
Annotation:1
d1
d1_log
Annotation:9
d9
d9_log

### CODE REVIEW

**Review by birm — CHANGES_REQUESTED** [review]
Can we call this new field something a little more reasable, like viewerState?

Maybe you want to save it somewhere inside, so that each drawn object has its own x,y,zoom, even if they're grouped together? I noted some issues when testing this in https://github.com/camicroscope/caMicroscope/issues/1013

**Comment by YukaUU** [comment]
> Can we call this new field something a little more reasable, like viewerState?

OK! 
 I change new field name.


> Maybe you want to save it somewhere inside, so that each drawn object has its own x,y,zoom, even if they're grouped together? I noted some issues when testing this in #1013

That's right.
I will fix it.

**Comment by YukaUU** [comment]
@birm 
I changed new field name from states to viewerStates.https://github.com/camicroscope/caMicroscope/pull/1011#:~:text=will%20fix%20it.-,change%20name%20from%20states%20to%20viewerStates,-10a8801

Test

1. Show x, y and zooming in pop panel
image

2.log (send data to DB)
change_states_viewerStates

3.log:DB collection:Mark
chabge_states_viewerStates

**Comment by YukaUU** [comment]
@birm

> Maybe you want to save it somewhere inside, so that each drawn object has its own x,y,zoom, even if they're grouped together? I noted some issues when testing this in https://github.com/camicroscope/caMicroscope/issues/1013

Not all annotations saved have (accurate) info saved #1013https://github.com/camicroscope/caMicroscope/issues/1013#issue-2414149542

I  fixed it!

**Test**
1. Annotation: Multi mode
Annotation: Multi mode

2. log (send data to DB)
image

3.log:DB collection:Mark
image

4. Check pop up panel
image
image
image
image
image

**Comment by YukaUU** [comment]
Delete viewerStates 
Past PR did not support Multi Mode.　https://github.com/camicroscope/caMicroscope/pull/966
I fixed it https://github.com/camicroscope/caMicroscope/pull/1011#issuecomment-2236869813
This commit was also deleted.https://github.com/camicroscope/caMicroscope/pull/1011#issuecomment-2236230522

**Test**
1. Annotation: Single mode
image
log:DB collection:Mark
image

2.Annotation: Multi mode
image
log:DB collection:Mark
image

File: core/extension/openseadragon-canvas-draw-overlay.js [hunk 1]
```diff
@@ -1023,6 +1024,11 @@
       if (this.drawMode == 'point') {
         this._current_path_ = {
           type: 'Feature',
+          viewerStates: {
+            x: StatesHelper.getCurrentStates(true).x,
+            y: StatesHelper.getCurrentStates(true).y,
+            z: StatesHelper.getCurrentStates(true).y
```
Comment (birm) [inline core/extension/openseadragon-canvas-draw-overlay.js:1030]:
```suggestion
            z: StatesHelper.getCurrentStates(true).z
```
I'm pretty sure this is why this is happening:
Screenshot 2024-07-19 at 10 49 08 AM
Screenshot 2024-07-19 at 10 49 02 AM

**Review by birm — CHANGES_REQUESTED** [review]
Very close; just one hopefully quick thing I noticed!

File: core/extension/openseadragon-canvas-draw-overlay.js (hunk 1 above)
Comment (YukaUU) [reply core/extension/openseadragon-canvas-draw-overlay.js:1030]: I also noticed a mistake in my local and was fixing the same area.
Thank you!!

**Comment by YukaUU** [comment]
@birm 
Refactored to remove unnecessary code, etc
I resolved all comments  and issue.
（https://github.com/camicroscope/caMicroscope/pull/1011#pullrequestreview-2183556879 , https://github.com/camicroscope/caMicroscope/issues/1013)
Please check it
### Test
Visualization Graph
image
log (send data to DB)
image
log:DB collection:Mark
image
Pop up panel
image
image
image
image
image

**Review by birm — APPROVED** [review]
Great work, and good fixes!

**Pull request merged**